src/i18n/en-US/discussions.ts

```diff
@@ -0,0 +1,98 @@
+const discussions = {
+  // TODO: We need to make a decision on how to structure these translations:
+  //         - Do we want a generic i18n file at all?
+  //         - Should some board specific stuff be stored in their respective i18n files? (e.g. name of the board, participants, role type translations, etc. in grbReview.ts)
+  //         - I tried to make the general section as generic as possible (obviously lol) but some of it may make the frontend overly complicated, open to any recommendations
+
+  general: {
+    cancel: 'Cancel', // TODO: this is in other i18n files, move to general.ts?
+    label: 'Discussions',
+    discussion: 'Discussion',
+    mostRecentActivity: 'Most recent activity',
+    newTopics: '{{count}} new discussion topic',
+    newTopics_plural: '{{count}} new discussion topics',
+    discussedTopics: '{{count}} discussion with replies',
+    discussedTopics_plural: '{{count}} discussions with replies',
+    fieldsMarkedRequired:
+      'Fields marked with an asterisk (<asterisk />) are required.', // TODO: this is in other i18n files, move to general.ts?
+    discussionsWithoutReplies: '{{count}} discussion without replies',
+    discussionsWithoutReplies_plural: '{{count}} discussions without replies',
+    readMore: 'Read more', // TODO: this is in other i18n files, move to general.ts?
+    repliesInDiscussion: '{{count}} reply in this discussion',
+    repliesInDiscussion_plural: '{{count}} replies in this discussion',
+    repliesCount: '{{count}} reply',
+    repliesCount_plural: '{{count}} replies',
+    reply: 'Reply',
+    lastReply: 'Last reply {{date}} at {{time}}',
+    hideReplies: 'Hide replies',
+    showReplies: 'Show replies',
+
+    startNewDiscussion: 'Start a new discussion',
+
+    view: 'View', // TODO: this is in other i18n files, move to general.ts?
+    viewDiscussionBoard: 'View discussion board',
+
+    viewMore: 'View more {{type}}',
+    viewLess: 'View less {{type}}',
+
+    alerts: {
+      noDiscussionsStarted:
+        'There are no discussions yet. When a discussion topic is started, it will appear here.',
+      noDiscussionsRepliedTo:
+        'There are no discussions yet. When a discussion topic is replied to, it will appear here.',
+      noDiscussionsStartButton:
+        'There are not yet any discussions. <button>Start a discussion</button>.',
+      replyError:
+        'There was an issue with adding your reply, please try again.',
+      replySuccess: 'Success! Your reply has been added.',
+      saveDiscussion:
+        'When you save your discussion, the selected team(s) and individual(s) will be notified via email.',
```

Comment:
is it worth it putting information into the UI about how we deduplicate emails?

Reply:
That would probably be a question for Zoe!

Reply:
I think it's probably also still accurate enough here. I wouldn't assume, as a user, that there's multiple email types that need de-duplicating.

I think just keeping it simple (as it is) lets us get across the msesage of "if you tag someone, they'll get a heads up"

Reply:
(otherwise it might just be kinda verbose)